recipes-ofdpa/ofdpa/ofdpa_3.0.5.0-EA5.bb

```diff
@@ -18,6 +18,8 @@ RDEPENDS_${PN} += " ${@bb.utils.contains('OFDPA_SWITCH_SUPPORT', 'BCM56770', '${
 # Nightly packages are regenerated regularily
 BB_STRICT_CHECKSUM = "0"
 
+# FEEDDOMAIN, FEEDURIPREFIX usually pulled in from
```

Comment:
```suggestion
# FEEDDOMAIN, FEEDURIPREFIX by default pulled in from
```

Reply:
meta-ofdpa has no idea where the variable is coming from, or what the default could be. It is determined by whoever is using meta-ofdpa. Even meta-switch doesn't know about meta-ofdpa or that it is defining a value for meta-ofdpa. If you use the bisdn-linux repo to build, then it brings meta-ofdpa and meta-switch together. I have no strong opinions on this, but that's the reason I went with "usually" instead of "by default". If you still prefer "by default", I'll make the change, np.

Reply:
Thanks, that is an improvement. Tweaked slightly and pushed.